week-2-practise/level-3/2-validation.js

```diff
@@ -6,7 +6,38 @@
 
 var excludedNums = [6, 14, 91, 111];
 
-function validate(num) {}
+function isNumber(num) {
+  if (typeof num === "number") {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isAnInteger(num) {
+  if (num % 1 === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isDifferent(num) {
+  if (!excludedNums.includes(num)) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function validate(num) {
```

Comment:
Good stuff!
As Ahmed noted elsewhere, it's possible to avoid `if/else` statements if your functions are returning booleans. Can you think of how to do that?